developer-docs/plugin-guide.md

```diff
@@ -38,19 +38,45 @@ These general steps are required if you want to make your own plugin and use it
   + Write your plugin
 
 #### Load your Plugin automatically
-With Pupil v0.6 we introduce a plugin auto-loader. It works when running from either source or application bundle! There is no need to put your plugin into the directories mentioned above. Instead:
-
- + In `~/pupil_capture_settings` or `~/pupil_player_settings` (depending on the plugin application) create a folder called `plugins`
- + Move your plugin source code file into `plugins`
- + If your plugin is defined by multiple files inside a directory, move this directory into the `plugins` dir <sup>1</sup>
- + On start-up Pupil will search this folder, and import and add all user plugins into the plugin drop-down menu 
+With Pupil v0.6 we introduce a plugin auto-loader. It works when running from either source or application bundle! There is no need to put your plugin into the directories mentioned above. The next steps should be followed instead:
+ 
+ + Choose which app should run your plugin and find its `plugins` folder. This folder should be automaticly created by the correspondent app during start up in `~/pupil_capture_settings`, for Pupil Capture, and in `~/pupil_player_settings` for Pupil Player.
+ + Move your plugin/source code file into the `plugins` folder. If your plugin is defined by multiple files inside a directory, move this directory into the `plugins` folder instead <sup>1</sup>
+ + On start-up Pupil will search for this folder, will import and add all valid plugins into the plugin drop-down menu. If your plugin is a calibration plugin (i.e., it inherits from the Calibration_Plugin base class), it will appear in the calibration plugins drop-down menu.
```

Comment:
Edit: 

Pupil will find your `plugins` on startup and will add all valid plugins to the plugin dropdown menu. If your plugin is a calibration plugin (i.e. it inherits from the Calibration_Plugin base class), then it will appear in the calibration drop down menu.